[CI] DocsClientYamlTestSuiteIT test {yaml=reference/cat/line_121} failing

Looks like sometimes `store.size`/`pri.store.size` for `my-index-000001` is lower than for `my-index-000002` so the test fail (order is different)

**Build scan:**
https://gradle-enterprise.elastic.co/s/4vi7aap7nrpd6/tests/:docs:integTest/org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT/test%20%7Byaml=reference%2Fcat%2Fline_121%7D

**Reproduction line:**
`./gradlew ':docs:integTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/cat/line_121}" -Dtests.seed=DD42A9BC027B51D -Dtests.security.manager=true -Dtests.locale=ar-DZ -Dtests.timezone=America/Inuvik -Druntime.java=8`

**Applicable branches:**
7.x, master

**Reproduces locally?:**
No

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT&tests.test=test%20%7Byaml%3Dreference/cat/line_121%7D

**Failure excerpt:**
```
java.lang.AssertionError: Failure at [reference/cat:2513]: field [$body] was expected to match the provided regex but didn't
Expected: health \s+status \s+index            \s+uuid                   \s+pri \s+rep \s+docs.count \s+docs.deleted \s+store.size \s+pri.store.size\s* yellow \s+open   \s+my-index-000001  \s+.+   \s+1   \s+1       \s+1200            \s+0      \s+\d+         \s+\d+\s* green  \s+open   \s+my-index-000002  \s+.+   \s+1   \s+0          \s+0            \s+0        \s+\d+          \s+\d+\s*
     but: was "health status index           uuid                   pri rep docs.count docs.deleted store.size pri.store.size\ngreen  open   my-index-000002 TweyyXYvThmXIXza6yXpSw   1   0          0            0        208            208\nyellow open   my-index-000001 ILsKjfI0RkekftwujU5W-g   1   1       1200            0        171            171\n"

  at __randomizedtesting.SeedInfo.seed([DD42A9BC027B51D:858015416EDBD8E5]:0)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:451)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:428)
  at sun.reflect.GeneratedMethodAccessor16.invoke(null:-1)
  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:498)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1750)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:938)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:974)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:988)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:817)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:468)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:947)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:832)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:883)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:894)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
  at java.lang.Thread.run(Thread.java:748)

  Caused by: java.lang.AssertionError: field [$body] was expected to match the provided regex but didn't
  Expected: health \s+status \s+index            \s+uuid                   \s+pri \s+rep \s+docs.count \s+docs.deleted \s+store.size \s+pri.store.size\s* yellow \s+open   \s+my-index-000001  \s+.+   \s+1   \s+1       \s+1200            \s+0      \s+\d+         \s+\d+\s* green  \s+open   \s+my-index-000002  \s+.+   \s+1   \s+0          \s+0            \s+0        \s+\d+          \s+\d+\s*
       but: was "health status index           uuid                   pri rep docs.count docs.deleted store.size pri.store.size\ngreen  open   my-index-000002 TweyyXYvThmXIXza6yXpSw   1   0          0            0        208            208\nyellow open   my-index-000001 ILsKjfI0RkekftwujU5W-g   1   1       1200            0        171            171\n"

    at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
    at org.junit.Assert.assertThat(Assert.java:956)
    at org.elasticsearch.test.rest.yaml.section.MatchAssertion.doAssert(MatchAssertion.java:57)
    at org.elasticsearch.test.rest.yaml.section.Assertion.execute(Assertion.java:65)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:444)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:428)
    at sun.reflect.GeneratedMethodAccessor16.invoke(null:-1)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1750)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:938)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:974)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:988)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
    at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
    at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
    at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:817)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:468)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:947)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:832)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:883)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:894)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
    at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
    at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
    at java.lang.Thread.run(Thread.java:748)

```

--- a/docs/reference/cat.asciidoc
+++ b/docs/reference/cat.asciidoc
@@ -33,17 +33,17 @@ Each of the commands accepts a query string parameter `v` to turn on
 verbose output. For example:
 
 [source,console]
---------------------------------------------------
-GET /_cat/master?v=true
---------------------------------------------------
+----
+GET _cat/master?v=true
+----
 
 Might respond with:
 
 [source,txt]
---------------------------------------------------
+----
 id                     host      ip        node
 u_n93zwxThWHi1PDBJAGAg 127.0.0.1 127.0.0.1 u_n93zw
---------------------------------------------------
+----
 // TESTRESPONSE[s/u_n93zw(xThWHi1PDBJAGAg)?/.+/ non_json]
 
 [discrete]
@@ -54,19 +54,19 @@ Each of the commands accepts a query string parameter `help` which will
 output its available columns. For example:
 
 [source,console]
---------------------------------------------------
-GET /_cat/master?help
---------------------------------------------------
+----
+GET _cat/master?help
+----
 
 Might respond with:
 
 [source,txt]
---------------------------------------------------
+----
 id   |   | node id
 host | h | host name
 ip   |   | ip address
 node | n | node name
---------------------------------------------------
+----
 // TESTRESPONSE[s/[|]/[|]/ non_json]
 
 NOTE: `help` is not supported if any optional url parameter is used.
@@ -82,16 +82,16 @@ Each of the commands accepts a query string parameter `h` which forces
 only those columns to appear. For example:
 
 [source,console]
---------------------------------------------------
+----
 GET /_cat/nodes?h=ip,port,heapPercent,name
---------------------------------------------------
+----
 
 Responds with:
 
 [source,txt]
---------------------------------------------------
+----
 127.0.0.1 9300 27 sLBaIGK
---------------------------------------------------
+----
 // TESTRESPONSE[s/9300 27 sLBaIGK/\\d+ \\d+ .+/ non_json]
 
 You can also request multiple columns using simple wildcards like
@@ -118,20 +118,21 @@ by shard storage descending order and then index name in ascending order.
 . The `v` (verbose) parameter to include column headings in the response.
 
 [source,console]
---------------------------------------------------
+----
 GET /_cat/indices?bytes=b&s=store.size:desc,index:asc&v=true
---------------------------------------------------
+----
 // TEST[setup:my_index_huge]
 // TEST[s/^/PUT my-index-000002\n{"settings": {"number_of_replicas": 0}}\n/]
+// TEST[s/s=store\.size\:desc,index\:asc/s=index:asc/]
 
 The API returns the following response:
 
 [source,txt]
---------------------------------------------------
+----
 health status index            uuid                   pri rep docs.count docs.deleted store.size pri.store.size
 yellow open   my-index-000001  u8FNjxh8Rfy_awN11oDKYQ   1   1       1200            0      72171         72171
 green  open   my-index-000002  nYFWZEO7TUiOjLQXBaYJpA   1   0          0            0        230          230
---------------------------------------------------
+----
 // TESTRESPONSE[s/72171|230/\\d+/]
 // TESTRESPONSE[s/u8FNjxh8Rfy_awN11oDKYQ|nYFWZEO7TUiOjLQXBaYJpA/.+/ non_json]
 
@@ -145,7 +146,7 @@ If you want to change the <<byte-units,byte units>>, use `bytes` parameter.
 ==== Response as text, json, smile, yaml or cbor
 
 [source,sh]
---------------------------------------------------
+----
 % curl 'localhost:9200/_cat/indices?format=json&pretty'
 [
   {
@@ -160,7 +161,7 @@ If you want to change the <<byte-units,byte units>>, use `bytes` parameter.
     "store.size": "650b"
   }
 ]
---------------------------------------------------
+----
 // NOTCONSOLE
 
 Currently supported formats (for the `?format=` parameter):
@@ -175,7 +176,7 @@ All formats above are supported, the GET parameter takes precedence over the hea
 For example:
 
 [source,sh]
---------------------------------------------------
+----
 % curl '192.168.56.10:9200/_cat/indices?pretty' -H "Accept: application/json"
 [
   {
@@ -190,7 +191,7 @@ For example:
     "store.size": "650b"
   }
 ]
---------------------------------------------------
+----
 // NOTCONSOLE
 
 [discrete]
@@ -207,21 +208,20 @@ For example, with a sort string `s=column1,column2:desc,column3`, the table will
 sorted in ascending order by column1, in descending order by column2, and in ascending
 order by column3.
 
-[source,sh]
---------------------------------------------------
+[source,console]
+----
 GET _cat/templates?v=true&s=order:desc,index_patterns
---------------------------------------------------
-//CONSOLE
+----
 
 returns:
 
 [source,txt]
---------------------------------------------------
+----
 name                  index_patterns order version
 pizza_pepperoni       [*pepperoni*]  2
 sushi_california_roll [*avocado*]    1     1
 pizza_hawaiian        [*pineapples*] 1
---------------------------------------------------
+----
 
 include::cat/alias.asciidoc[]
 

--- a/docs/reference/cat.asciidoc
+++ b/docs/reference/cat.asciidoc
@@ -83,7 +83,7 @@ only those columns to appear. For example:
 
 [source,console]
 ----
-GET /_cat/nodes?h=ip,port,heapPercent,name
+GET _cat/nodes?h=ip,port,heapPercent,name
 ----
 
 Responds with:
@@ -119,7 +119,7 @@ by shard storage descending order and then index name in ascending order.
 
 [source,console]
 ----
-GET /_cat/indices?bytes=b&s=store.size:desc,index:asc&v=true
+GET _cat/indices?bytes=b&s=store.size:desc,index:asc&v=true
 ----
 // TEST[setup:my_index_huge]
 // TEST[s/^/PUT my-index-000002\n{"settings": {"number_of_replicas": 0}}\n/]

